cypress/e2e/chartdata.cy.js

```diff
@@ -0,0 +1,31 @@
+describe('Chart Data Test', () => {
+  it('Chart data is maintained across pages', () => {
+    //visit the line page
+    cy.visit('/line.html');
+
+    //Enter the chart title
+    cy.get('#chart-title-input').type('title')
+    //Enter the x and y labels
+    cy.get('#x-label-input').type('x');
+    cy.get('#y-label-input').type('y');
+
+    //Enter the x and y values 
+    cy.get('[data-testid="x-input"]').type('7');
+    cy.get('[data-testid="y-input"]').type('5');
+    cy.get('#generate-chart-btn').click();
+
+    //Navigate to the Scatter page
+    cy.get('nav').contains('Scatter').click();
+    //click the generate button
+    cy.get('#generate-chart-btn').click();
+
+    //Check that the chart is visible
+    cy.get('#chart-display img').should('be.visible');
+    //Check that the chart title is maintained
+    cy.get('#chart-title-input').should('have.value', 'title');
+
```

Comment:
These assertions are great! Could you please also add a couple to verify that the X and Y labels are preserved after the navigation?

Reply:
Thanks for expanding the test here! I think I was a bit unclear in my first note, could you add something like `cy.findByLabelText('X label')...` to ensure that the axes' labels are preserved after navigation?

Reply:
I think we're still on different pages here. I'm thinking this test should include something like lines 44-45 of `reopen chart.cy.js`:
```js
     cy.findByLabelText('X label').should('have.value', 'x');
     cy.findByLabelText('Y label').should('have.value', 'y');
```

Is there a reason those weren't included?

Reply:
Just forgot to add those lines to the rest of the files! Adding them now.